axelor-gdpr/src/main/resources/views/GDPRRequest.xml

```diff
@@ -173,4 +174,12 @@
     <field name="accessResponse.sendingDateTime" expr="eval: __datetime__"/>
   </action-record>
 
+  <action-attrs name="action-gdpr-request-attrs-hide-cancel">
+    <attribute for="statusSelect" name="selection-in" expr="eval: [0,1,2]"/>
+    <attribute for="cancelSendResponseButton" name="hidden"
+      expr="eval: (statusSelect == 1 || statusSelect == 2) &amp;&amp; typeSelect == 1"/>
+    <attribute for="cancelButton" name="hidden"
+      expr="eval: (statusSelect == 1 || statusSelect == 2) &amp;&amp; typeSelect == 1"/>
```

Comment:
Don't hard-code selection item values but use the constants defined in the repo instead

Reply:
incomplete, please fix review and fix the `if` attributes and all the `attribute` elements of this actions